2024-10-04/README.md

```diff
@@ -0,0 +1,174 @@
+     hosted at Mozilla Berlin.
+   * Social Web Foundation: Evan is now working with SWF with the goal of
+     improving the network and also the user experience. Not yet a W3C member
+     but will be soon, for now Evan is participating as individual.
+3. Social Web CG Charter discussion
+    * Group has had great momentum in the last year but we need to make sure we have a charter
+    * https://github.com/swicg/potential-charters/pull/1
+    * Evan: when we started the CG we still had the Social Web Working Group,
+      when we started the CG we said "it will work [handwave] like the WG does".
+      But that was 5 or 6 years ago. My question is what kind of timeframe are
+      we hoping for in this discussion and approval?
+        * Dmitri: anywhere between 2 to 4 weeks is my propsal, what do people think?
+        * (discussion)
+        * CG charters do not require W3C approval (probably)
+        * __Proposal__: accept
+          [PR #1](https://github.com/swicg/potential-charters/pull/1) 
+          as the basis for collaboration on building a charter
+          * Outcome: **Resolved**
+          * Bumblefudge (in chat): i think the important part is to make
+            significant progress EACH OF THE 4 WEEKS on the various PRs so that
+            next week we MIGHT be ready to ratify haha
+            who wants me to PR on their behalf can just tag me in their issue
+    * Evan: how are the WG and CG charters going to be related
+      * Bengo: if we can use the CG charter as a platform which we can agree on
+        high level WG goals, it will make everything easier
+      * Bob Wyman (in chat):PRs work for coders, but given that this is
+         "social" web, perhaps we should use Issues, which are easier for
+         non-coders. Or, provide a "PRs for Dummy's".
+          * Tantek (in chat): or file an issue?
+          * Bumblefudge (in chat): yeah issues are pretty user-friendly. anyone
+    * Emelia: how does this relate to the question of who the editor(s) for the
+      spec is(are)?
+      * Dmitri: the typical editor/chair selection for a WG is W3C staff
+        appoints chairs, chairs appoint editor, based on community feedback
+        generally. I would like to make this explicit rather than implicit for
+        both the CG and the WG via elections. W3C has now set up a voting
+        mechanism for members and held elections for one WG so at least there is
+        precedent. For editors maybe a similar voting mechanism?
+      * Emelia: on the voting procedure, I was involved in the Solid one. There
+        was a comment from someone on the forums who was unhappy with how the
+        Solid election went, there was a perception that Inrupt carried lots of
+        weight and won a lot of board seats. Virginia ran as an independent
+        without Inrupt supporting her so it's important to note that while she
+        is Inrupt affiliated, she did not represent inrupt. We need to be clear
```

Comment:
Her affiliation has changed. I don't know the exact dates, but I'm pretty sure it happened before this meeting, and it _may_ have happened before that election. 
```suggestion
        was Inrupt affiliated, she did not represent Inrupt. We need to be clear
```